### PR TITLE
[JENKINS-37098] Since tag cleanup

### PR DESCRIPTION
**Pull request by jglick**
Amends #2490.

@reviewbybees esp. @ndeloof, @oleg-nenashev

### CODE REVIEW

**Comment by ndeloof** [comment]
:+1:

File: core/src/main/java/hudson/util/io/RewindableRotatingFileOutputStream.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
  * {@link ReopenableFileOutputStream} that does log rotation upon rewind.
  *
  * @author Kohsuke Kawaguchi
- * @since 1.416
+ * @since 2.18
```
Comment (daniel-beck) [inline core/src/main/java/hudson/util/io/RewindableRotatingFileOutputStream.java]: Also wrong author and wrong copyright date.

**Comment by oleg-nenashev** [comment]
👍

File: core/src/main/java/hudson/util/io/RewindableRotatingFileOutputStream.java (hunk 1 above)
Comment (jglick) [inline core/src/main/java/hudson/util/io/RewindableRotatingFileOutputStream.java]: True, but I am just solving one thing at a time here.

**Pull request merged**